packages/libssp.rb

```diff
@@ -0,0 +1,43 @@
+require 'package'
+
+class Libssp < Package
+  description 'Libssp is a part of the GCC toolkit.'
+  homepage 'https://gcc.gnu.org/'
+  version '10.3.0'
+  license 'GPL-3, LGPL-3, libgcc, FDL-1.2'
```

Comment:
How can so many licenses exist for this?  It's confusing.

Reply:
These are all the licenses for GCC, I don't know libssp's licenses specifically